crates/sui-types/src/unit_tests/messages_tests.rs

```diff
@@ -588,66 +600,6 @@ fn test_user_signature_committed_in_signed_transactions() {
     assert_ne!(signed_tx_a, signed_tx_b)
 }
 
-#[test]
-fn test_user_signature_committed_in_checkpoints() {
```

Comment:
this test is trivially needed since the transaction digest is now derived from the same transaction data even if the signer is different. hence the checkpoint derived from the same tx digest is the same too. 

added a scenario for test_user_signature_committed_in_transactions which covers the case more meaningfully